infrastructure/aws/lambda/Dockerfile

```diff
@@ -10,13 +10,16 @@ COPY README.md README.md
 COPY titiler/ titiler/
 
 # Install dependencies
+
+# Install system dependencies to compile (numexpr)
+RUN yum install -y gcc-c++
 # HACK: aiobotocore has a tight botocore dependency
 # https://github.com/aio-libs/aiobotocore/issues/862
 # and becaise we NEED to remove both boto3 and botocore to save space for the package
 # we have to force using old package version that seems `almost` compatible with Lambda env botocore
 # https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html
 RUN python -m pip install --upgrade pip
-RUN python -m pip install . "mangum>=0.10.0" "botocore==1.29.76" "aiobotocore==2.5.0" -t /asset --no-binary pydantic
+RUN python -m pip install . "mangum>=0.10.0" "botocore==1.35.36" "aiobotocore==2.15.2" -t /asset --no-binary pydantic
```

Comment:
are you sure those version are compatible with the boto3 version which are in AWS Lambda image?

Reply:
Good catch! I found the versions of boto3 and botocore in the 3.10 Lambda image and matched them up.